modules/plugin/imagemosaic/src/main/java/org/geotools/gce/imagemosaic/GranuleDescriptor.java

```diff
@@ -97,31 +98,33 @@
 
 /**
  * A granuleDescriptor is a single piece of the mosaic, with its own overviews and everything.
- * 
+ * <p>
  * <p>
  * This class is responsible for caching the various size of the different levels of each single granuleDescriptor since computing them each time is
  * expensive (opening a file, looking for a reader, parsing metadata,etc...).
- * 
+ * <p>
  * <p>
  * Right now we are making the assumption that a single granuleDescriptor is made a by a single file with embedded overviews, either explicit or
  * intrinsic through wavelets like MrSID, ECW or JPEG2000.
- * 
+ *
  * @author Simone Giannecchini, GeoSolutions S.A.S.
  * @author Stefan Alfons Krueger (alfonx), Wikisquare.de : Support for jar:file:foo.jar/bar.properties URLs
- * @since 2.5.5
- *
- *
  * @source $URL$
+ * @since 2.5.5
  */
 public class GranuleDescriptor {
 
-    /** Logger. */
+    /**
+     * Logger.
+     */
     private final static Logger LOGGER = org.geotools.util.logging.Logging
             .getLogger(GranuleDescriptor.class);
 
     private static final String AUXFILE_EXT = ".aux.xml";
 
-    /** Hints to use for avoiding to search for the ImageMosaic format */
+    /**
```

Comment:
Looks like there still are lots of javadoc related pure reformats in this pull request. Please do not use automatic formatting, it makes bug fixes backports harder down the road :-)